specificatie/genereervariant/openapi.yaml

```diff
@@ -2354,7 +2381,7 @@ components:
               deel van is.
             items:
               type: string
-              example: "0226100000008856"
+              example: '["0226100000008856"]'
```

Comment:
Dit is niet correct. Hier staat dat een pandIdentificatie de string `["0226100000008856"]` kan zijn. Volgens mij moet het als volgt worden gedefinieerd:

```
            items:
              type: string
            example:
            - "0226100000008856"
```